workflows/argo/offline-diags.yaml

```diff
@@ -33,6 +37,17 @@ spec:
             value: /secret/gcp-credentials/key.json
           - name: CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
             value: /secret/gcp-credentials/key.json
+          - name: WANDB_ENTITY
+            value: ai2cm
+          - name: WANDB_PROJECT
+            value: "{{inputs.parameters.wandb-project}}"
+          - name: WANDB_TAGS
+            value: "{{inputs.parameters.wandb-tags}}"
+          - name: WANDB_RUN_GROUP
```

Comment:
This is called `wandb-group` in the microphysics argo workflow: https://github.com/ai2cm/fv3net/blob/c09ea6bebe68cac49f5a30ab52751ea43fb66d99/projects/microphysics/argo/argo.yaml#L30

I like your name better, but I think it would be best to be consistent with the existing precedent for now.  We can change both to wandb-run-group later on.

Reply:
Also changed train-diags-prog and training templates to "run-group"